### DISCUSSION TITLE
Next 13.2.0 dev build error - Cannot access default.then on the server

### DISCUSSION THREAD

**Discussion by kavitang** [vercel/next.js#46318] [Help]
I just upgraded to Next 13.2.0 and now my development environment refuses to work. Any navigation to my main pages results in the following error

```
 Error: Cannot access default.then on the server. You cannot dot into a client module from a server component. You can only pass the imported name through.
```
I'm not able to find any information on the error. When I reverted back to 13.1.6 everything worked fine.

I'm using the new `app` directory.

**Comment by st11x** [comment]
I have the same problem. It looks like they killed off the head.ts|js prematurely. If you have that file, you will get that error.
